recipes/sqlcipher/all/patches/Makefile.in-v4.4.0.patch

```diff
@@ -1,13 +1,22 @@
 diff --git a/Makefile.in b/Makefile.in
-index d8f5a62..0f082e9 100644
+index d8f5a62..57b54d5 100644
 --- a/Makefile.in
 +++ b/Makefile.in
+@@ -647,7 +647,7 @@ DBFUZZ_OPT =
+ # This is the default Makefile target.  The objects listed here
+ # are what get build when you type just "make" with no arguments.
+ #
+-all:	sqlite3.h libsqlcipher.la sqlcipher$(TEXE) $(HAVE_TCL:1=libtclsqlite3.la)
++all:	sqlite3.h libsqlcipher.la
+ 
+ Makefile: $(TOP)/Makefile.in
+ 	./config.status
 @@ -1445,9 +1445,8 @@ lib_install:	libsqlcipher.la
  	$(INSTALL) -d $(DESTDIR)$(libdir)
  	$(LTINSTALL) libsqlcipher.la $(DESTDIR)$(libdir)
  
 -install:	sqlcipher$(TEXE) lib_install sqlite3.h sqlcipher.pc ${HAVE_TCL:1=tcl_install}
-+install:	lib_install sqlite3.h
++install:	lib_install sqlite3.h sqlcipher.pc
```

Comment:
is it required? we anyway remove pkg-config files, and use conan-generated only

Reply:
No, but I want to avoid as much as possible to modify upstream Malefile.in. I prefer to remove files after installation, it's a good hint for recipe maintainers that there is a pkg_config file to emulate.

Reply:
but if you want less modifications in Makefile.in... but never mind, up to you. we may probably avoid patch altogether, but if it's a useful hint, why not?
WFYT @uilianries ?

Reply:
The patch was already there (it was removing more dependencies of install target), I had to modify it so that we can call `make all` without errors